src/handlers/motions/motionStaked/motionStaked.ts

```diff
@@ -36,10 +39,54 @@ export default async (event: ContractEvent): Promise<void> => {
   if (stakedMotion) {
     const {
       id,
-      motionData: { usersStakes },
+      motionData: { usersStakes, events: updatedEvents },
       motionData,
     } = stakedMotion;
 
+    if (vote.eq(0) && !updatedEvents.find(({ name }) => name === 'ObjectionRaised')) {
+      updatedEvents.push(
+        {
+          name: 'ObjectionRaised',
+          transactionHash,
+          logIndex,
+          initiatorAddress: staker,
+        },
+      );
+    }
+
+    updatedEvents.push(
+      {
+        name: 'MotionStaked',
+        transactionHash,
+        logIndex,
+        initiatorAddress: staker,
+        vote: vote.toString(),
+        amount: amount.toString(),
+      },
+    );
+
+    if (requiredStake.eq(BigNumber.from(motionStakes.raw.yay))) {
+      const eventName = !updatedEvents.find(({ name }) => name === 'ObjectionRaised') ? 'MotionFullyStaked' : 'MotionFullyStakedAfterObjection';
+      updatedEvents.push(
+        {
+          name: eventName,
+          transactionHash,
+          logIndex,
+          initiatorAddress: staker,
+        },
+      );
+    }
+    if (requiredStake.eq(BigNumber.from(motionStakes.raw.nay))) {
```

Comment:
You don't need to create a BigNumber for the motion stake since `requiredStake` is already a BigNumber and BigNumber methods can handle strings